bundle/README.md

```diff
@@ -152,3 +152,111 @@ To configure `bookinfo` with a gateway using `Gateway API`:
     $ echo "http://${GATEWAY_URL}/productpage"
     ```
 
+## Istio Addons Integrations
+
+Istio can be integrated with other software to provide additional functionality (More information can be found in: https://istio.io/latest/docs/ops/integrations/). The Sail Operator provides support for the following integrations:
+
+### Prometheus
+
+`Prometheus` is an open-source systems monitoring and alerting toolkit. You can use `Prometheus` with the Sail Operator to keep an eye on how healthy Istio and the apps in the service mesh are, for more information, see [Prometheus](https://istio.io/latest/docs/ops/integrations/prometheus/). Take into count that you will need to have installed the istio control plane before to start with this integration.
+
+To install Prometheus you will need to follow this steps:
+
+1. Deploy the operator:
+
+    ```sh
+    $ oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/addons/prometheus.yaml
+    ```
+2. Access to `Prometheus`console:
+
+* Option 1: 
+
+    * Expose the `Prometheus` service
```

Comment:
To make it a bit more clear what's happening, perhaps say "Expose the `Prometheus` service **externally**
(comment applies to the other addons as well)